tests/uc-dev_test_prod_setup.tfvars

```diff
@@ -7,7 +7,7 @@ region        = "us10"
 # Project specific configuration
 # ------------------------------------------------------------------------------------------------------
 landscapes       = ["DEV", "TST", "PRD"]
-unit             = "Integration Test - dev test prod"
+unit             = "Integration Test"
```

Comment:
This is just "Integration Test" ;-) Shouldn't there also be dev test prod or something like that

Reply:
Makes sense and I'll have to update the naming convention for that in the respective module, as the names of the unit are restricted.